### PR TITLE
Updated "configuration.rst" in Italian

### PR DESCRIPTION
**Pull request by matteocoder**
| Q             | A
| ------------- | ---
| Bug fix?      | yes/no
| New feature?  | yes/no
| BC breaks?    | yes/no
| Deprecations? | yes/no
| Tests pass?   | yes/no
| Documentation | yes
| Translation   | yes
| Fixed tickets | #...
| License       | MIT

### CODE REVIEW

File: docs/it/user/configuration.rst [hunk 1]
```diff
@@ -47,9 +47,11 @@ Potete cambiare il vostro nome, il vostro indirizzo email e abilitare l'``Autent
 Autenticazione a due fattori (2FA)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-L'autenticazione a due fattori (conosciuta anche come 2FA) é una tecnologia brevettata nel 1984 che offre l'identificazione degli utenti tramite una combinazione di due componenti differenti.
+    L'autenticazione a due fattori (conosciuta anche come 2FA) é una tecnologia brevettata nel 1984 che offre l'identificazione degli    utenti tramite una combinazione di due componenti differenti.
```
Comment (j0k3r) [inline docs/it/user/configuration.rst]: Could you please remove these tabs?

Comment (matteocoder) [inline docs/it/user/configuration.rst]: Ok no problem, I saw them in the English version too, so I though it was correct to insert them.

**Comment by matteocoder** [comment]
I think now it should be ok.

**Comment by nicosomb** [comment]
As @nicofrand said here https://github.com/wallabag/wallabag/pull/2905, we already have a PR for that. 
I close this one. Thank you for the :it: translation!

**Pull request closed without merging**